src/components/phone-input/PhoneInput.tsx

```diff
@@ -1,24 +1,27 @@
 import {useState} from 'react';
 import {PhoneInputSectionItem, Section} from '@atb/components/sections';
 import {ActivityIndicator, StyleProp, View, ViewStyle} from 'react-native';
-import {LoginTexts, useTranslation} from '@atb/translations';
+import {PhoneInputTexts, useTranslation} from '@atb/translations';
 import {StyleSheet, useTheme} from '@atb/theme';
 import {PhoneSignInErrorCode} from '@atb/auth';
 import {StaticColorByType, getStaticColor} from '@atb/theme/colors';
 import {MessageInfoBox} from '@atb/components/message-info-box';
 import {Button} from '@atb/components/button';
 import phone from 'phone';
 import {SvgProps} from 'react-native-svg';
+import {GetAccountByNumberErrorCode} from '@atb/on-behalf-of/types';
+
+type PhoneInputErrorCode = PhoneSignInErrorCode | GetAccountByNumberErrorCode;
 
 type Props = {
   style?: StyleProp<ViewStyle>;
   submitButtonText: string;
   submitButtonTestId: string;
-  onSubmitPromise: (
+  phoneNumberValidation: (
```

Comment:
Kinda hard to understand that it is a function based on the parameter name when it is just a noun. It is easier to understand when using "on" as prefix as that is a pretty standard convention. Or something like `validatePhoneNumber` is also clearer that is a function, when it starts with a verb.

Reply:
renamed into ´validatePhoneNumber´